sdk/key_vault/src/key.rs

```diff
@@ -239,6 +239,54 @@ impl Display for SignatureAlgorithm {
     }
 }
 
+#[derive(Debug, Deserialize, Getters)]
+#[getset(get = "pub")]
+pub struct DecryptResult {
+    aad: Option<String>,
+    iv: Option<String>,
```

Comment:
Even though `aad` and `iv` shouldn't be defined here, they are base64url-encoded and should be exposed as binary, i.e. `Option<Vec<u8>>`